controller_manager:
  ros__parameters:
    update_rate: 50  # Hz

    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

joint_trajectory_controller:
  ros__parameters:
    joints:
      - servo0
      - servo1
      - servo2
      - servo3
      - servo4
      - left_gripper_joint
      - right_gripper_joint
    interface_name: position
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity